R/uvhydrograph-styles.R

```diff
@@ -6,6 +6,16 @@ getUvStyle <- function(data, info, correctionLabels, plotName, dataSides, dataLi
   comp_lbl <- info$comp_UV_lbl
   comp_type <- info$comp_UV_type
   legend.name <- data[[1]]$legend.name
+  corrArrowPositions <- list()
+  plotEndDate <- tail(info$plotDates,1) + hours(23) + minutes(45)
```

Comment:
actually, I think this is different. `toEndOfMonth` will put it  on the last day of that month, but that might not be the last day of the data. It might be worth adding `plotStartDate` and `plotEndDate` to `primaryInfo` so that you don't have to do this calc in more than one location. Then you can just change anything that uses it to `primaryInfo$plotEndDate`